mkt/submit/tests/test_views.py

```diff
@@ -1011,7 +1010,8 @@ def test_done(self):
         eq_(res.status_code, 200)
 
 
-class TestNextSteps(amo.tests.TestCase):
+class TestNextSteps(TestCase):
+    ##(ej)## Delete below now?
```

Comment:
I removed my comment and left @ngokevin 's for now.  If a decision is made prior to this being ready to merge I'll change but doesn't seem like a blocker.